Recommand recent genomes (#238)
nf-core/nf-co.re: markdown/developers/guidelines.md

  * Nearly all nf-core pipelines use a conda `env` script to list their software requirements.
    The pipeline Docker images are then built using this, meaning that with a single file your pipeline can support nextflow users running with conda, docker or singularity.
  * The [nf-core template](/tools#creating-a-new-workflow) comes with all required code to support this setup.

> * All pipelines should use by default a recent reference genomes (_ie_ `GRCh38` for human...)

Reviewer: ```suggestion
* All pipelines should attempt to use a recent reference genome draft(_ie_ `GRCh38` for human...)
```
